cmd/ebrelayer/main.go

```diff
@@ -165,35 +167,40 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 	}
 
 	// Validate and parse arguments
-	networkDescriptor, err := strconv.Atoi(args[0])
+	networkDescriptor, err := cmd.Flags().GetInt32(networkDescriptorFlag)
 	if err != nil {
-		return errors.Errorf("%s is invalid network id", args[0])
+		return err
```

Comment:
Is the new error message descriptive or generic? Does it specify that the network id flag was missing/invalid?

Reply:
it is a generic error description.

Reply:
Can we make this specific? The problem when using this CLI is when a flag is incorrect/missing generic error messages make it a real pain to debug. What happens if the flag is missing here does it return an error?

Reply:
OK, will add specific error info in the command line.